### PR TITLE
Known issue for CCO Manual mode

### PR DESCRIPTION
**Pull request by codyhoag**

### CODE REVIEW

**Comment by codyhoag** [comment]
@yunjiang29 can you confirm? Thanks!

**Comment by yunjiang29** [comment]
@codyhoag thanks for update, LGTM

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -226,6 +226,11 @@ new modes that can be configured:
 * Passthrough
 * Manual
 
+[IMPORTANT]
+====
+Azure and GCP do not support the Manual mode configuration using the `install-config.yaml` file due to a known issue found in link:https://bugzilla.redhat.com/show_bug.cgi?id=1884691[BZ#1884691].
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:231]:
```suggestion
Azure and GCP do not support the Manual mode configuration by using the `install-config.yaml` file due to a known issue found in link:https://bugzilla.redhat.com/show_bug.cgi?id=1884691[BZ#1884691].
```

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -2043,6 +2048,8 @@ virtual media, the provisioning network must be turned off, or the BMCs must be
 routed to the provisioning network as a prerequisite.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1872787[*BZ#1872787*])
 
+* The {product-title} installation program does not support the Manual mode configuration using the `install-config.yaml` file on GCP and Azure due to a known issue. As a workaround, you must manually insert a ConfigMap into the manifest directory during the manifest generation stage of the cluster installation process. You can reference xref:../installing/installing_azure/manually-creating-iam-azure.adoc#manually-creating-iam-azure[Manually creating IAM for Azure] or xref:../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-creating-iam-gcp[Manually creating IAM for GCP] for more details. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1884691[*BZ#1884691*])
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:2051]: It sounds like this configuration method is a legit option, so I suggest not calling it a workaround.

```suggestion
* The {product-title} installation program does not support the Manual mode configuration by using the `install-config.yaml` file on GCP and Azure due to a known issue. Instead, you must manually insert a ConfigMap into the manifest directory during the manifest generation stage of the cluster installation process as documented in xref:../installing/installing_azure/manually-creating-iam-azure.adoc#manually-creating-iam-azure[Manually creating IAM for Azure] and xref:../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-creating-iam-gcp[Manually creating IAM for GCP]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1884691[*BZ#1884691*])
```

**Pull request merged**